doc/source/release/1.16.0-notes.rst

```diff
@@ -88,43 +124,832 @@ New features
 
 ``scipy.stats`` improvements
 ============================
+- Added `scipy.stats.quantile`, an array API compatible function for quantile
+  estimation.
+- Extended `scipy.stats.make_distribution` to:
+
+  - work with existing discrete distributions and
+  - facilitate the creation of custom distributions
+
+  in the new random variable infrastructure.
+
+- Added `scipy.stats.Binomial`.
+- Added ``equal_var`` keyword to:
+  - `scipy.stats.tukey_hsd` (enables the Games-Howell test) and
+  - `scipy.stats.f_oneway` (enables Welch ANOVA).
+- Improved moment calculation for `scipy.stats.gennorm`.
+- Added native vectorization to `scipy.stats.mode` for faster batch calculation.
+- Added ``axis``, ``nan_policy``, and ``keepdims`` to `scipy.stats.power_divergence`,
+  `scipy.stats.chisquare`, `scipy.stats.pointbiserialr`, `scipy.stats.kendalltau`,
+  `scipy.stats.weightedtau`, `scipy.stats.theilslopes`, `scipy.stats.siegelslopes`,
+  and `scipy.stats.boxcox_llf`.
+- Improved speed of `scipy.stats.special_ortho_group`.
+-
 
+**************************
+Array API Standard Support
+**************************
 
+Experimental support for array libraries other than NumPy has been added to
+existing sub-packages in recent versions of SciPy. Please consider testing
+these features by setting an environment variable ``SCIPY_ARRAY_API=1`` and
+providing PyTorch, JAX, or CuPy arrays as array arguments. Many functions
+in `scipy.stats`, `scipy.special`, `scipy.optimize`, and `scipy.constants`
+now provide tables documenting compatible array and device types as well as
+support for lazy arrays and JIT compilation. New features with support and
+old features with support added for SciPy 1.16.0 include:
```

Comment:
There was discussion in https://github.com/tylerjereddy/scipy/pull/83 about whether we should re-list parts of SciPy that previously had array API support, but didn't have i.e., JAX or Dask support.

Or whether it would be preferable to only include parts of SciPy for which basic array API support previously wasn't present at all.

There are some verbosity tradeoffs to consider perhaps.

Reply:
I'd not go overboard, but add something like: "In addition, some functionality that already supported other array libraries in previous releases extended that support - most commonly by adding JAX and Dask support in addition to PyTorch and CuPy."